Makefile

```diff
@@ -46,12 +46,15 @@ install-service: check-prev-proxies
 
 	install -D -m 755 trigger-pac4cli $(DESTDIR)/etc/NetworkManager/dispatcher.d/trigger-pac4cli
 	install -D -m 755 pac4cli.sh $(DESTDIR)/etc/profile.d/pac4cli-proxy.sh
+	install -D -m 644 pac4cli.config $(DESTDIR)/etc/pac4cli/pac4cli.config
 
 install-bin:
 	install -D -m 755 main.py $(DESTDIR)$(bindir)/pac4cli
 	@sed -i -e '1s+@PYTHON@+'$(PYTHON)'+' $(DESTDIR)$(bindir)/pac4cli
 
 	install -D -m 644 pac4cli.py $(DESTDIR)$(pythonsitedir)/pac4cli.py
+	install -D -m 644 wpad.py $(DESTDIR)$(pythonsitedir)/wpad.py
+	install -D -m 644 servicemanager.py $(DESTDIR)$(pythonsitedir)/servicemanager.py
```

Comment:
Cool, we're on the same page then.

>  But since my knowledge in Python is still pretty limited, I opted to ask you to review the changes in small chunks and fail early.

With that in mind, I went over the code again to see if there was anything python specific I should teach/warn you about, but your code actually already looks very Pythonic. You're a natural :)

As for the helper modules, I don't really know of a way to make modules completely private. I think the common thing to do is to just namespace them:

 - make a directory `$(pythonsitedir)/pac4cli`
 - make an (empty) file `$(pythonsitedir)/pac4cli/__init__.py` just to signal to python that this directory is a module.
 - put the files like `wpad.py` in the directory.

Now a statement like `import pac4cli.wpad` should work. Additionally, any functions you put in `__init__.py` can be imported through `import pac4cli`.

Hope that helps, in case you decide to go that route! I don't mind either way.